backend/spec/support/request_spec_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module RequestSpecHelper
+  def auth_user
```

Comment:
Let's receive user as an argument (instead of creating one) in case we will need to have custom settings for this record.
And `auth_user` name for this method is not really says what it is doing. This is more about authentication header itself

I.e. I'd better call and use it like this
```ruby
let(:headers) { auth_header_for(user) }
subject(:request) { post '/v1/api', params: params, headers: headers }
```